ambry-tools/src/main/java/com/github/ambry/clustermap/HelixVcrPopulateTool.java

```diff
@@ -60,11 +60,10 @@ public static void main(String[] args) throws IOException {
     OptionSet options = parser.parse(args);
 
     String[] destZkAndCluster = options.valueOf(destOpt).split(HelixVcrUtil.SEPARATOR);
-    if (destZkAndCluster.length != 3) {
-      errorAndExit("dest argument must have form 'zkString/namespace/clusterName'");
-    }
-    String destZkString = destZkAndCluster[0] + HelixVcrUtil.SEPARATOR + destZkAndCluster[1];
-    String destClusterName = destZkAndCluster[2];
+    // string could be zk/namespace/clusterName or zk/clusterName
+    int lastSeparator = destZkAndCluster[0].lastIndexOf(SEPARATOR);
+    String destZkString = destZkAndCluster[0].substring(0, lastSeparator);
+    String destClusterName = destZkAndCluster[0].substring(lastSeparator+1);
```

Comment:
This is better, but one bug here, it should be
```
String destZkAndCluster = options.valueOf(destOpt);
int lastSep = destZkAndCluster.lastIndexOf(SEPARATOR);
String destZkString = destZkAndCluster.substring(0, lastSep);
String destClusterName = destZkAndCluster.substring(lastSep + 1);
```

Reply:
good catch !